drivers/wifi/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig WIFI
+        bool "Wi-Fi Drivers"
+
+if WIFI
+
+# TODO: FIX THIS
+module = WIFI_1
```

Comment:
This is not working (its same Zephyr), throws below error:

```
warning: the default selection WIFI_LOG_LEVEL_DEFAULT (defined at subsys/net/Kconfig.template.log_config.net:41, subsys/net/Kconfig.template.log_config.net:41) of <choice> (defined at subsys/net/Kconfig.template.log_config.net:7) is not contained in the choice
```
Line #41 is within a choice, for now changing name works, will fix this before merge, but just putting it out there.